### PR TITLE
reason to go to Zamboanga

### PR DESCRIPTION
**Pull request by ccaechu**
Lets go there

### CODE REVIEW

File: web_source/cities/asia/philippines/zamboanga.md [hunk 1]
```diff
@@ -4,12 +4,17 @@ Country: Philippines (Asia)
...
+- Chavacano: Ciudad de Zamboanga
+- Tausug: Dāira sin Sambuangan
+- Filipino: Lungsod ng Zamboanga
+- Cebuano: Dakbayan sa Zamboanga
 
 ### What we would like to see there
 
-FIXME
+Zamboanga Pink Sand Escape 
+![image](https://github.com/UCL-ARC-Workshops/202405-arcgit-workshop/assets/132667328/93fa56ac-81bc-44fe-86ea-2619e328188c)
```
Comment (vyorkeedwards) [inline web_source/cities/asia/philippines/zamboanga.md]: Can you add more information to the image label?
```suggestion
![image of pink sands](https://github.com/UCL-ARC-Workshops/202405-arcgit-workshop/assets/132667328/93fa56ac-81bc-44fe-86ea-2619e328188c)
```

**Review by vyorkeedwards — CHANGES_REQUESTED** [review]
Suggest updating image label, otherwise looks great

**Comment by ccaechu** [comment]
I think I revise it.....thanks

**Review by vyorkeedwards — APPROVED** [review]
Looks great!

**Pull request merged**